ZipBuilder/Sources/ZipBuilder/main.swift

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+// Get the launch arguments, parsed by user defaults.
+let args = LaunchArgs()
+
+// Clear the cache if requested.
```

Comment:
Do we really still want the cache?

Reply:
It certainly doesn't make sense on the CI, but it did bring down local build times from ~18 minutes to ~3 minutes for me when iterating. Let's chat about it!